twitchio/ext/routines/__init__.py

```diff
@@ -99,6 +99,7 @@ def __init__(
 
         self._args: tuple | None = None
         self._kwargs: dict | None = None
+        self.next_event_time:datetime.datetime|None = None #type : ignore
```

Comment:
This is improperly formatted, and why is this type ignore necessary?

```suggestion
        self.next_event_time: datetime.datetime | None = None  #type : ignore
```

Reply:
copy/pasta error re: type ignore.